portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Makes the clicked header link have the class "active" and removes the "active" class from the other header links
+ */
+function setActiveTab(btn) {
+    //Gets all tabs with the class name "active" (there should only be 1)
+    const activeTabs = document.getElementsByClassName('active');
+
+    //Remove the "active" class from that element
+    activeTabs[0].classList.remove('active'); 
+
+    //Add "active" class to the clicked element
+    btn.classList.add('active'); 
```

Comment:
nit: These two comments are pretty self explanatory - you can delete them to avoid over commenting.